src/bin/vip-settings.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
```

Comment:
I hate these doc blocks 😆 . Wish I could kill them all with fire, but we at least blast away the unused ones :)

Reply:
@WPprodigy time to drop Calypso from lint settings :)